Microservices/ServiceRegistryMicroservice/src/main/resources/db/changelog-master.xml

```diff
@@ -0,0 +1,175 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+
+<!--    We will have a parent directory -->
+    <include file="com/example/news/news.changelog.sql"/>
+    <include file="com/example/directory/directory.changelog.sql"/>
+
+    <changeSet author="Killian" id="0">
```

Comment:
A suggestion I'd make is to have all the changesets merged into ID 0, because this is the first change (or creation) applied to the DB which should be under 1 changeset (or commit as you could think)